x-pack/plugins/rule_registry/README.md

```diff
@@ -2,62 +2,129 @@
 
 The rule registry plugin aims to make it easy for rule type producers to have their rules produce the data that they need to build rich experiences on top of a unified experience, without the risk of mapping conflicts.
 
-A rule registry creates a template, an ILM policy, and an alias. The template mappings can be configured. It also injects a client scoped to these indices.
+The plugin installs default component templates and a default lifecycle policy that rule type producers can use to create index templates.
 
-It also supports inheritance, which means that producers can create a registry specific to their solution or rule type, and specify additional mappings to be used.
+It also exposes a rule data client that will create or update the index stream that rules will write data to. It will not do so on plugin setup or start, but only when data is written.
 
-The rule registry plugin creates a root rule registry, with the mappings defined needed to create a unified experience. Rule type producers can use the plugin to access the root rule registry, and create their own registry that branches off of the root rule registry. The rule registry client sees data from its own registry, and all registries that branches off of it. It does not see data from its parents.
+## Configuration
 
-## Enabling writing
-
-Set
+By default, these indices will be prefixed with `.alerts`. To change this, for instance to support legacy multitenancy, set the following configuration option:
 
 ```yaml
-xpack.ruleRegistry.unsafe.write.enabled: true
+xpack.ruleRegistry.index: '.kibana-alerts'
 ```
 
-in your Kibana configuration to allow the Rule Registry to write events to the alert indices.
+To disable writing entirely:
+
+```yaml
+xpack.ruleRegistry.write.enabled: false
```

Comment:
This will also need to be done here https://github.com/dgieselaar/kibana/blob/120521feb6b3359613601a10af329e0aca100645/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker#L279 and here in the readme https://github.com/dgieselaar/kibana/blob/120521feb6b3359613601a10af329e0aca100645/x-pack/plugins/observability/README.md#L22-L23

I'm ok with this being in follow-up if you want to get this merged.

Reply:
i think the first one is generated, I've opened an issue for the second one: https://github.com/elastic/kibana/issues/100039.

Reply:
It says it's generated but it's not.

Reply:
☠️